docs/3.0. Discovery.md

```diff
@@ -28,14 +28,29 @@ protocols.
 Clients and Resource Servers MUST support discovering the Authorization Server through use of unicast DNS-SD service
 discovery, as described in [RFC 6763][RFC-6763].
 
+### Authorization Server Metadata Endpoint
+
+The Server Metadata endpoint is described in [RFC 8414][RFC-8414]. The "/.well-known" URI suffix of "oauth-
+authorization-server" MUST be used to identify the Authorization Server metadata endpoint. The value of the
+Authorization Server metadata URL should take the form:
+
+```
+{ api_proto }://{ hostname }:{ port }/.well-known/oauth-authorization-server[/{ api_label }]
```

Comment:
Could you indicate that the `api_` placeholders come from DNS-SD TXT records described further down?